editoast/src/views/infra/mod.rs

```diff
@@ -60,6 +64,32 @@ pub fn routes() -> impl HttpServiceFactory {
         )
 }
 
+#[derive(Debug, Error, EditoastError)]
+#[editoast_error(base_id = "infra")]
+pub enum InfraApiError {
+    /// Couldn't find the infra with the given id
+    #[error("Infra '{infra_id}', could not be found")]
+    #[editoast_error(status = 404)]
+    NotFound { infra_id: i64 },
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct InfraForm {
+    pub name: String,
+}
+
+impl From<InfraForm> for Infra {
+    fn from(infra: InfraForm) -> Self {
+        Infra {
+            name: Some(infra.name),
+            owner: Some(uuid!("00000000-0000-0000-0000-000000000000")),
```

Comment:
We can use https://docs.rs/uuid/latest/uuid/struct.Uuid.html#method.nil

```suggestion
            owner: Some(Uuid::nil()),
```